packages/safe-tools-client/app/components/future-payments-list/index.gts

```diff
@@ -31,33 +32,45 @@ export default class FuturePaymentsList extends Component<Signature> {
   @service declare network: NetworkService;
   // @ts-expect-error loading services/scheduled-payments
   @service('scheduled-payments') declare scheduledPaymentsService: ScheduledPaymentsService;
+  @service declare date: DateService;
+  
+  get now(): Date {
+    return new Date(Number(this.date.now()));
```

Comment:
We have a [clock service](https://github.com/cardstack/cardstack/blob/main/packages/hub/services/clock.ts) in the hub. Maybe follow that approach so that our time travel techniques are consistent